### PR TITLE
[RT-391] add adr documentation

### PR DESCRIPTION
**Pull request by rpr8895**
### What is the goal?

Use a common PR template

 ### References
* **Issue:** https://sequra.atlassian.net/browse/RT-422

 ### Definition of done

Use a common PR template

 ### How is it being implemented?

- Add CODEOWNERS file
- Add docs files
- Improve README

 ### Opportunistic refactorings

no

 ### Caveats

no

 ### How is it tested?

manual

### CODE REVIEW

File: docs/decisions/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Architectural design records
+The idea is to provide a basic integrated ADR generation for #{your project}
```
Comment (albertvinyes) [inline docs/decisions/README.md:2]: I think we can replace #{your project} with Factiva API Clint

File: docs/decisions/index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Architectural Decision Log
+
+This log lists the architectural decisions for [project name].
```
Comment (albertvinyes) [inline docs/decisions/index.md:3]: Same here

**Review by albertvinyes — COMMENTED** [review]
We can also update the PR template in `.github`, the current template is a bit old

File: docs/decisions/README.md (hunk 1 above)
Comment (rpr8895) [reply docs/decisions/README.md:2]: nice catch!

**Review by albertvinyes — APPROVED** [review]

**Review by sinfantes4 — APPROVED** [review]

**Pull request merged**